ui/job-view/headerbars/ActiveFilters.jsx

```diff
@@ -134,15 +134,16 @@ export default class ActiveFilters extends React.Component {
           <div>
             <form className="form-inline">
               <div className="form-group input-group-sm new-filter-input">
-                <label className="sr-only" htmlFor="job-filter-field">
+                <span className="sr-only" id="field-input-label">
```

Comment:
I assume sr-only means this is visually hidden. In that case, I wonder whether it'd be simpler to just have aria-label="Field" on the select.

Reply:
Correct about the ``sr-only``.  Supposed to mean "Screen Reader Only".  I like your suggestion better.  :)  Done.